src/main/java/seedu/address/model/student/admin/AdditionalDetail.java

```diff
@@ -48,7 +48,7 @@ public int hashCode() {
      * Format state as text for viewing.
      */
     public String toString() {
-        return '[' + detail + ']';
+        return "[" + detail + "] ";
```

Comment:
Why is there a need for this? I believe it was the same way in the `Tag` implementation in AB3, since when these are put together there is another module with its method to put a space between tags.

Reply:
I kind of just followed what was already in place. I also felt that the brackets just makes additional details easier to organize visually for now as multiple additional details could look very messy and hard to separate (for easy viewing) without the brackets.

Reply:
Oh I see, I will look into what this other module is. As of now, the space was added just to space out the different additional details when it is being displayed in the console (when a student has been successfully added).